compiler/control/OMROptions.cpp

```diff
@@ -2813,6 +2815,12 @@ OMR::Options::jitPreProcess()
       _maxSpreadCountLoopless = TR_MAX_SPREAD_COUNT_LOOPLESS;
       _maxSpreadCountLoopy = TR_MAX_SPREAD_COUNT_LOOPY;
 
+      // Set the default value for _edoRecompSizeThreshold before the
```

Comment:
Are these two defaults chosen to be the same value because it is expected to be overridden/changed anyway in a downstream project ?

Reply:
The values are the same to match the old behaviour. The old code did not specialize based on start-up phase. 
The values may change in the future based on extensive experiments.